### DISCUSSION TITLE
Stepper color support

### DISCUSSION THREAD

**Discussion by suchintan** [mantinedev/mantine#2838] [Feature requests]
Hey @rtivital 

I was messing around with the stepper and I realized that I can't pass in custom hex values for the `color` property 
https://mantine.dev/core/stepper/

Do you know if it would be possible to support custom hex values instead of just `MantineColor`?

**Comment by rtivital** [comment]
Yes, we will do that in one of the next releases

**Comment by suchintan** [comment]
Nice!! Looking forward to it!
